etc/deploy/aws.sh

```diff
@@ -305,6 +297,21 @@ if [[ "${USE_CLOUDFRONT}" == "true" ]]; then
   echo "Created cloudfront distribution with ID: ${CLOUDFRONT_ID}"
 fi
 
+# Put the cluster address in the pachyderm config
+config_path="${HOME}/.pachyderm/config.json"
+[[ -d "${HOME}/.pachyderm" ]] || mkdir "${HOME}/.pachyderm"
+[[ -e "${config_path}" ]] || {
+  echo '{}' >"${config_path}"
+  chmod 777 "${config_path}"
+}
+tmpfile="$(mktemp -p .)"
+cp "${config_path}" "${tmpfile}"
+jq --monochrome-output \
+  ".v1.pachd_address=\"${K8S_MASTER_DOMAIN}:30650\"" \
+  "${tmpfile}" \
+  >"${config_path}"
+rm "${tmpfile}"
+
 # Must echo ID at end, for etc/testing/deploy/aws.sh
-echo "Cluster created:"
+echo "Cluster created. Cluster address has been written to \$HOME/.pachyderm/config:"
```

Comment:
Maybe move the "Cluster Created" message up to the point where the cluster has actually been created, before outputting the address?